src/tools/rest-backend/rest-backend-config.json

```diff
@@ -0,0 +1,14 @@
+{
+	"service": {
+		"api": "http",
+		"port": 8080,
+		"ip": "0.0.0.0"
+	},
+	"http" : {
+		"script_names": ["/"]
+	},
+	"api_specification": {
```

Comment:
With elektrifying cppcms this would not matter anymore. It is okay to have it in the run-time config!